Sources/Conduit/Networking/Serialization/HTTPRequestSerializer.swift

```diff
@@ -109,7 +109,7 @@ open class HTTPRequestSerializer: RequestSerializer {
     static let httpMethodsWithNoBody: Set<HTTPRequestBuilder.Method> = [.GET, .HEAD]
 
     /// Required request headers for HTTP transport according to the W3 spec
-    open static let defaultHTTPHeaders: [(String, String)] = {
+    public static let defaultHTTPHeaders: [(String, String)] = {
```

Comment:
This change should be noted in the CHANGELOG

Reply:
Done.